releasenotes/notes/0.18/higher-order-spsa-f2eddb44b00b82e0.yaml

```
@@ -0,0 +1,24 @@
+---
+features:
+  - |
+    A new optimizer class, :class:`~qiskit.algorithms.optimizers.QNSPSA`,
+    has been added to the :mod:`qiskit.algorithms.optimizers` module. This
+    class implements the
+    `Quantum Natural SPSA (QN-SPSA) <https://arxiv.org/abs/2103.09232>`__
+    algorithm, which a generalization of 2-SPSA algorithm, estimates the
+    Quantum Fisher Information Matrix instead of a the Hessian to obtain a
+    stochastic estimate of the Quantum Natural Gradient. For examples on
+    how to use this new optimizer refer to the
+    :class:`~qiskit.algorithms.optimizers.QNSPSA` class documentation.
+  - |
+    A new kwarg, ``second_order``, has been added to the constructor
+    of the :class:`~qiskit.algorithms.optimizers.SPSA` class in the
+    :mod:`qiskit.algorithms.optimizers` module. When set to ``True`` this
+    enables using
+    `second-order SPSA <https://ieeexplore.ieee.org/document/657661>`__.
+    Second order SPSA is an extension of the ordinary SPSA algorithm that
```

Since we refer to 2-SPSA in the QNSPSA reno, it might be nice to use this abbreviation here too 🙂 
```suggestion
    Second order SPSA, or 2-SPSA, is an extension of the ordinary SPSA algorithm that
```